config/initializers/app_keys.rb.example

```diff
@@ -1,4 +1,9 @@
 # environemnt variables for setup_mail.rb
 ENV["email_user_name"] = "PUT YOUR USERNAME HERE"
 ENV["email_password"]  = "PUT YOUR PASSWORD HERE"
-ENV["bing_key"] = "PUT YOUR BING KEY HERE"
+
+ENV["GOOGLE_API_KEY"] = "API KEY HERE"
+ENV["GOOGLE_SEARCH_CX"] = "SEARCH CX KEY HERE"
+
+ENV['aws_access_key_id']     = "AWS ACCESS KEY HERE"
+ENV['aws_secret_access_key'] = "AWS SECRET KEY HERE"
```

Comment:
For the same consistency it's best to use either all caps or all lower case. I think all caps makes more sense in this case.